WordPress/Classes/Categories/NSURLRequest+Helpers.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+
+@interface NSURLRequest (Helpers)
```

Comment:
So I'll explain the reasoning behind my comment, but keep in mind the current implementation is OK and you can opt to keep it.

So the reason why I consider subclassing a better option here is because of single-responsibility.  The class `NSURLRequest` has the single responsibility of providing us with a generic interface to make URL requests, without caring about what those requests are used for.  By adding a method to that class that knows what our requests look like, it feels to me like we're stepping a bit outside of that class' domain.

However, with that in mind, feel free to leave it as it is.  Just my 2 cents.